posts/upcoming-lts-12-ghc-8-4-3.md

```diff
@@ -0,0 +1,20 @@
+---
+author: Alexey Zabelin
+title: Upcoming stackage LTS 12 snapshot with ghc-8.4.3
+description: Upcoming stackage LTS 12 snapshot with ghc-8.4.3
+timestamp: 2018-06-17T00:01:00Z
+---
+
+After a [recent discussion](https://mail.haskell.org/pipermail/haskell-cafe/2018-June/129259.html)
+on the Haskell-cafe mailing list, the stackage curator team has decided
+to start the preparations for an LTS major bump which will include
+ghc-8.4.3.
+
+We'd like to give everyone enough time to prepare, which is why the
+planned release date is **two weeks** from the date of this post.
+
+As always, if you'd like to get your package added to an LTS snapshot,
```

Comment:
Maybe worth clarifying that, for now, getting the packages into nightly will ensure they make it into LTS 12. It's only after the LTS 12 release happens that they'd need to open an issue against lts-haskell.

Reply:
Good point, thanks!